src/style/App.css

```diff
@@ -39,3 +47,8 @@
   font-size: 14px;
   font-weight: 500;
 }
+
+.Asset-item {
+  display: block;
+  width: 100%;
+}
```

Comment:
Can we move this to components/AssetListItem.css and import it from AssetListItem.js? Also, I don't see where this class is actually applied to an element. Is it implicit?